content/platform/data-services-cloud-console/home.md

```diff
@@ -1,177 +1,42 @@
 
-![DSCC Automation](/img/dscc-api-value.png "API for Automation at Scale")
+### Introduction to Data Services Cloud Console
 
-## A Public REST API which is based on the OpenAPI 3.0
+To access the Data Service Cloud Console,  click on this URL [](https://common.cloud.hpe.com)<https://common.cloud.hpe.com> and sign-in using your HPE GreenLake Console's account. [](https://common.cloud.hpe.com)
 
-Hewlett Packard Enterprise (HPE) offers the DSCC public REST API to provide the agility as previously mentioned . It is specified based on the OpenAPI format version 3 ([OpenAPI 3.0 information](https://swagger.io/blog/news/whats-new-in-openapi-3-0/)).  The specification defines a standard, language-agnostic interface to the RESTful API allowing the clients (both human and computer) to consume capabilities of DSCC services efficiently. The API definition is available for download in OpenAPI 3 YAML or JSON format at the link mentioned in the next paragraph.
+For an  overview of the Data Services Cloud Console and to get started with the Data services Cloud Console please click on this link <https://hpe.com/storage/dscc>. 
 
-Some of the advantages of distributing in OpenAPI 3.0 format:
+<iframe width="560" height="315" src="https://www.youtube.com/embed/AxUE89X3Sy0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-1. Updates to the API can be generated in a more agile manner where documentation is embedded, describing any endpoints, parameters, and more; such as contact information, license, terms of use. 
-2. Consumers of this API also gain the benefits of agility, by using the converter from openAPI yaml, or json to any programming language that is used as part of the their automation or CI/CD workflow. (Please check <https://openapi.tools> for more information for the API tools to generate client code)
+<iframe width="560" height="315" src="https://www.youtube.com/embed/lzOWapX0m5U" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
-## DSCC REST API Details
+### Data Services Cloud Console Public REST API
 
-Any one can download this OpenAPI Specification (OAS) v3 definition of the DSCC from the following: [Link to the API repository](https://console-us1.data.cloud.hpe.com/doc/api/v1/)
+Data Services Cloud Console public REST API provides a resource for customers who are looking to enhance their infrastructure management and data-ops using the programmatical extensions from Data Services Cloud Console.  More blog posts will be coming that will provide more information to help customers adopting this API, providing examples, code snippets and other helpful information
```

Comment:
Change "programmatical" to "programmatic"
Don't forget to add a period at the very end of this paragraph.